_includes/nav.html

```diff
@@ -1,23 +1,21 @@
-<!-- <nav class="">
-  <div class="container">
-
-
-
-
-    <a href="/" {% if page.url == "/" %}class="active"{% endif %}>Home</a>
-    |
-    <a href="/about" {% if page.url == "/about" %}class="active"{% endif %}>About</a>
-
-  </div>
-</nav> -->
+<<nav id="nav"  class="navbar navbar-expand-sm ">
+      <div class="container nav__container">
 
+        {% if page.url contains "bcjobs" %}
+          <a class="navbar-brand" href="https://www.bcjobs.ca/">
+        {% include svg-logo.html %}
+        </a>
+        {% elsif page.url contains "novascotiajobs" %} 
+         <a class="navbar-brand" href="https://www.novascotiajobs.ca/">
+          <img class="navbar-logo" src="/assets/img/logo-novascotiajobs.png">
+        </a>
+        {% elsif page.url contains "albertajobcentre" %} 
+        <a class="navbar-brand" href="https://www.albertajobcentre.ca/">
+          <img class="navbar-logo" src="/assets/img/logo-albertajobcentre.png">
+        </a>
+        {% endif %}
```

Comment:
There's a lot of repetition here. I suggest you define a `site_name` variable that will contain one of the following:
* bcjobs
* albertajobcentre
* novasotiajobs

One way to define `site_name` (but not the only way) is as follows:

```
{% assign site_name = page.url | remove_first: "/" | split: "/" | first %}
```

Then you can use it like this to avoid the repetition:

```
         <a class="navbar-brand" href="https://www.{{site_name}}.ca/">
          <img class="navbar-logo" src="/assets/img/logo-{{site_name}}.png">
        </a>
```

## Optional Reading
If you're curious about `{% assign site_name = page.url | remove_first: "/" | split: "/" | first %}`, please read on. If not, please ignore:
* Starts with `page.url`, which is something like `/albertajobcentre/50-off-month-one`
* Remove the first `/` and then splits it by `/`, which results in an array of 2 items:
  * albertajobcentre
  * 50-off-month-one
* Take the first item in the array, which is `albertajobcentre`.

Reply:
Great idea. I was using an SVG logo for BCJobs before, but switched it to a PNG to use this simpler approach. If I can get ahold of the vector files for Alberta, NS, I'll switch them all over to SVG using this same approach. 

Gotta love liquid! I guess its a matter of opinion/taste but I find the ability to do this on a single line in a "humanist" syntax, is clean and readable.